doc/manual_experimental.rst

```diff
@@ -17,6 +17,11 @@ Some of these are not covered by the ``.experimental`` pragma or
 one may want to use Nim libraries using these features without using them
 oneself.
 
+Whether or not an experimental feature is enabled can be checked in user code
+by checking if the symbol with the prefix ``nimHas`` and the name of the
+experimental feature is defined. For example, to check if ``implicitDeref`` is
+enabled, you can use the check ``defined(nimHasImplicitDeref)``.
```

Comment:
you should also clarify that `nimHasFoo` is used for things other than experimental flags; it's  used more generally to test whether compiler supports some feature `foo` (see condsyms.nim for all usages)

Reply:
Well I'm saying "if" here not "if and only if". Mentioning that it's used for other features would just confuse people more because what other features it's used for isn't documented.